src/Elmish.WPF/BindingInternal.fs

```diff
@@ -168,11 +164,9 @@ module internal BindingData =
       | OneWayToSourceData d -> OneWayToSourceData {
           Set = binaryHelper d.Set
         }
-      | OneWaySeqLazyData d -> OneWaySeqLazyData {
+      | OneWaySeqData d -> OneWaySeqData {
           Get = f >> d.Get
-          Map = d.Map
```

Comment:
I thought we were going to have issues here, but yeah I do see now how this is fully redundant with `Get`